partcad/src/partcad/part_factory_sweep.py

```
@@ -0,0 +1,178 @@
+from . import logging as pc_logging
+from .utils import resolve_resource_path
+
+
+class PartFactorySweep(PartFactory):
+    depth: float
+    source_project_name: str
+    source_sketch_name: str
+    source_sketch_spec: str
+    sketch: Sketch
+
+    def __init__(self, ctx, source_project, target_project, config):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)